### DISCUSSION TITLE
Question about Home tracker legality

### DISCUSSION THREAD

**Discussion by NinFanBoyFTW** [kwsch/PKHeX#4124] [Q&A] [answered]
Hi, I just wanted to ask about possible legality issues regarding pokemon that are transferred directly from save to save (Across generations on PKHeX, without Home tracker IDs. Obviously Home tracker IDs are created when the pokemon goes through Home, but is there a legality concern when it comes to say a pokemon from 4th gen or even 8th gen coming into the 9th gen saves without any Home tracker ID? Is that possible to have no Home tracker ID transferring those Pokémon legit, or would it be a red flag that PKHeX was used?

**Comment by kwsch** [comment]
> Is that possible to have no Home tracker ID transferring those Pokémon legit, or would it be a red flag that PKHeX was used?

Not possible without cheating. PKHeX has a setting that is default to "Fishy" for missing HOME Trackers when a Pokémon should have one. You can change it so it is truly strict instead of lenient for local play.

**Reply by NinFanBoyFTW** [reply]
Ok got it thanks for the clarification. Is a work around putting those pokemon that were transferred from PKHeX through Home and back so they get home tracker IDs? Just was not sure if pokemon home itself would flag anything as well

**Reply by Lusamine** [reply]
In many cases, HOME will error out because it recognizes that certain Pokemon cannot enter it from a particular game with missing data.
If your goal is playing by yourself with a Pokemon that's not going to leave your save file, you don't have to worry about a HOME tracker at all. If you're wanting to move it around to other games, the real solution is to transfer legitimately or find someone else to do it for you.

**Reply by NinFanBoyFTW** [reply]
So I just got around to testing everything and seeing what happens with Pokemon home so I confirmed what you said would happen. At that point I personally feel that these pokemon should be flagged as illegal by PKHeX if the are in a future generation without any pokemon home tracker ID. I'm sure there are plenty of other tiny examples of things in the past that have been flagged and created as a rule in PKHeX because legality is extremely important. I didn't think that the legality system was built into PKHeX for the purposes of local play, I would actually think that it would be built in specifically for the purposes of making sure that a mon is legit for online play/ safe with Pokemon servers and services.

I understand that you might say if you dig into the legality report this comes up as fishy but in my opinion these pokemon should be flagged as illegal or at least a different symbol to denote that they are legal in every other way except for a null home id. I know myself personally, I have always used PKHeX to transfer pokemon across generations because it makes everything seamless and simple. I'm sure others do the same, and I don't think it is good to inadvertantly create illegal mons that could have the ramifications of bans, or otherwise without flagging these mons the same way you would flag anything else that would make it impossible for a mon to appear legitimately on that save file. If people choose to use these mons after understanding that they are illegal in the eyes of pokemon home, then at least you can say they understand that risk

**Reply by NinFanBoyFTW** [reply]
I also think with the closure of the eshop, and looming threat of the pokemon bank service closing at some un named point in the future, people should be given as much information as possible about these mons so that they do not find out sometime much further down the line that their mons are illegal in the eyes of pokemon's servers when it is too late to simply boot up their 3DS and transfer everything legit.

**Reply by sora10pls** [reply]
> At that point I personally feel that these pokemon should be flagged as illegal by PKHeX if the are in a future generation without any pokemon home tracker ID.

Like you were already told, PKHeX has a setting you can adjust for HOME tracker validity; the default is fishy, since the intent is to cover every other legality check before checking for the presence of a tracker.

**Reply by NinFanBoyFTW** [reply]
No, I understand that, and I changed that on my end, but I don't think that is a good default to have. I don't know how many people out there understand these options or the possible ramifications with them, but I am sure that a ton of people just want to know if their pokemon are legit, period. It should be a conscious choice to understand when you turn that option to fishy, you are losing some of the visibility of legality because you do not care if those mons can be transferred through home, or appear 100% legit online.

**Reply by Lusamine** [reply, marked as answer]
It is impossible for the program to let you know if any Pokémon is "legit" (aka obtained in-game with no edits). It can only check if your Pokémon is legal (aka has possible traits).

That said, it is default because if you are using it to edit data for personal use, the preference is not to flag them because it won't matter. If you care about legality, you can opt in rather than force every user by default to care about a legality check that doesn't affect local, personal use.

**Reply by Doctor-Pokemon** [reply]
I **WAS** going to agree with NinFanBoyFTW, but Lusamine brings up a very good point...
Most people using PKHeX are using it for personal recreation & not to cheese the online filters.
So most users won't care about a "fishy" result as long as PKHeX will save the edited savegame.

So having the Invalid Home Tracker legality check default set to "Fishy" makes a lot more sense to me now...
If you care, you'll know what's wrong;
if you don't care, you can ignore it, and PKHeX will still save the edited savegame.

Thank you, Lusamine.
